src/wallet-instance-attestation/issuing.ts

```diff
@@ -38,7 +38,7 @@ export class Issuing {
       jti: `${uuid.v4()}`,
       type: "WalletInstanceAttestationRequest",
       cnf: {
-        jwk: publicKey,
+        jwk: fixBase64EncodingOnKey(publicKey),
```

Comment:
IMHO this parsing has to be done upstream by who generates the key in JWK format, so probably `io-react-native-crypto`

Reply:
ok